osioauth/tokens.go

```diff
@@ -0,0 +1,129 @@
+package osioauth
```

Comment:
you can call the package `token` instead of `osioauth/tokens`
( in general avoid `osio` in naming because its a jargon of our downstream name :) )

Reply:
done!